co-log/src/Colog/Message.hs

```diff
@@ -296,18 +303,49 @@ fmtRichMessageDefault RichMessage{..} = do
      <> thread
      <> msgText
 
-    showTime :: C.Time -> Text
-    showTime t =
-        square
-        $ toStrict
-        $ TB.toLazyText
-        $ C.builder_DmyHMS timePrecision datetimeFormat (C.timeToDatetime t)
-      where
-        timePrecision = C.SubsecondPrecisionFixed 3
-        datetimeFormat = C.DatetimeFormat (Just '-') (Just ' ') (Just ':')
-
-    showThreadId :: ThreadId -> Text
-    showThreadId = square . T.pack . show
+{- | Formats 'RichMessage' in the following way:
+
+@
+[Time] [SourceLocation] [ThreadId] \<Text message\>
+@
+
+__Examples:__
+
+@
+[03 05 2019 05:23:19.058] [Main.example#34] [ThreadId 11] app: First message...
+[03 05 2019 05:23:19.059] [Main.example#35] [ThreadId 11] app: Second message...
+@
+
+Practically, it formats a message as 'fmtRichMessageDefault' without the severity information.
+-}
+fmtSimpleRichMessageDefault :: MonadIO m => RichMessage m -> m Text
+fmtSimpleRichMessageDefault msg = fmtRichMessageCustom msg formatRichMessage
+   where
+    formatRichMessage :: Maybe ThreadId -> Maybe C.Time -> Message -> Text
+    formatRichMessage (maybe "" showThreadId -> thread) (maybe "" showTime -> time) Msg{..} =
+        time
+     <> showSourceLoc msgStack
+     <> thread
+     <> msgText
+
+fmtRichMessageCustom :: MonadIO m => RichMessage m -> (Maybe ThreadId -> Maybe C.Time -> Message -> Text) -> m Text
+fmtRichMessageCustom RichMessage{..} formatter = do
+    maybeThreadId  <- extractField $ TM.lookup @"threadId"  richMsgMap
+    maybePosixTime <- extractField $ TM.lookup @"posixTime" richMsgMap
+    pure $ formatter maybeThreadId maybePosixTime richMsgMsg
+
+showTime :: C.Time -> Text
+showTime t =
+            square
+          $ toStrict
+          $ TB.toLazyText
+          $ C.builder_DmyHMS timePrecision datetimeFormat (C.timeToDatetime t)
+        where
```

Comment:
`where` should have 2-space indentation and other functions should have 4-space indentation according to our style guide.